examples/hnas/example_mlp.py

```diff
@@ -0,0 +1,228 @@
+import autogluon.core as ag
```

Comment:
Add comment at the top of this file describing what this example is supposed to show. Also consider explicitly stating what auxiliary packages must be installed to run this example.

Reply:
ok added